src/components/DashboardPage.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {Card, Button, Container, Row, Col} from 'react-bootstrap';
+
+class Dashboard extends React.Component {
+  render() {
+    return (
+      <container>
+      <div>
+        <Container>
+          <Row>
+            <Col>
+      <Card className="card1">
+  <Card.Body>
+    <Button className="button" href="#" variant="primary">Add Recipe</Button>
+  </Card.Body>
+</Card>
+</Col>
+<Card className="card2">
+  <Card.Body>
+    <Button className="button"  href="#" variant="primary">View Recipes</Button>
+  </Card.Body>
+</Card>
```

Comment:
As each card on this page is the same just with different text you could use a loop where you just pass in the text to each card.